addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -15,7 +15,7 @@
                         threadLocalId="chatWindow.thread.localId"
                     />
                 </t>
-                <div class="o_ChatWindowHeader_item o_ChatWindowHeader_name" t-att-title="chatWindow.name">
+                <div class="o_ChatWindowHeader_item o_ChatWindowHeader_name" t-att-title="chatWindow.name" t-att-class="{'o-clickable': chatWindow.thread and chatWindow.thread.channel_type === 'chat' and chatWindow.thread.correspondent}" t-on-click="_onClickChatHeaderName">
```

Comment:
Don't check for `channel_type` at all, if the thread has a unique `correspondent` it makes sense to be able to open his profile. Just then maybe livechat should do an override if the correspondent is the public partner, because that one makes no sense. But when using livechat with a normal use it's fine to allow one profile.